### ISSUE TITLE
[🐛 Bug]: Personal resul activation does not work

### ISSUE THREAD

**Issue by KappaSpende** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#616]
### What went wrong?

When naming the device and toggling the "personal results" options on, nothing changes on the PC, and the same Error occurs. When checking my phone, the option is somehow toggled off again.

### What did you expect to happen?

Error message should vanish, and the Toggle on my Setting should still be turned on.

### Steps to reproduce the bug

See Text above. I did everything [this post](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/594) says

### App Version

v1.0.0-rc.2

### OS Version

Windows 10 Home 20H2

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by KappaSpende** [comment]
In addition, I did reinstall the whole thing. Didn't help.

**Comment by im-coder-lg** [comment]
Did you try the wiki guide or the gist guide I made?

**Comment by im-coder-lg** [comment]
> First, head to the Actions Console.  Locate and visit your present GAUDC project.
Go to the Develop section.
Hit the hamburger button on the left side of the Develop screen and go to Account Linking.
Now, enable Account linking by switching the toggle.
Choose yes for Account Creation.
Hit Google Sign-in as the Linking Type.

Try that. It might work.

**Comment by KappaSpende** [comment]
> Did you try the wiki guide or the gist guide I made?

Followed a tutorial, then repeated it following the wiki.

**Comment by im-coder-lg** [comment]
What tutorial?

**Comment by trodix** [comment]
This did not worked for me :/

**Comment by im-coder-lg** [comment]
Which tutorial did you use?

**Comment by trodix** [comment]
I followed this tutorial : 
https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Setup-Authentication-for-Google-Assistant-Unofficial-Desktop-Client

Then as mentioned on this post I enabled Account Linking as below :

![image](https://user-images.githubusercontent.com/34584317/148647298-f65abab4-866e-47bc-9c08-3eaa14baa781.png)

I did not received any notification on my phone

**Comment by im-coder-lg** [comment]
You must enable account linking first, then follow the steps.

**Comment by ratacuco** [comment]
Hey, I have fallowed every step and guide there is but it doesn't work for me, when I enable it in my phone the app crashes and resets the personal results setting

**Comment by im-coder-lg** [comment]
Actually, for @KappaSpende the bug is in-app. YOurs seem to be a mobile problem. Try using another Android.

**Comment by Abtaby** [comment]
Step 1: open google home.
Step 2: go to news section.
Step 3: ask for personal results on the device you installed the google client.
Step 4: click on allow and name the device.
Step 5: click on save and turn on personal results,
- It worked for me!

**Comment by xtcrefugee** [comment]
Abtaby's answer worked for me, I wasn't even getting any prompts on my phone to allow Assistant to provide personal answers for the Windows client. For some reason Google Home was needed to see the requests from this aplication.

**Comment by hanny00** [comment]
How can i enable personal results without having a phone?
I install the Appimage on a Linux Laptop with a second Google Account without a phone.
The App runs very well, thank you for your work

**Comment by im-coder-lg** [comment]
You need a phone or any Android VM. Only then Personal Results will work since you'll have to approve the request.

**Comment by Ahmed00010** [comment]
> Step 1: open google home. Step 2: go to news section. Step 3: ask for personal results on the device you installed the google client. Step 4: click on allow and name the device. Step 5: click on save and turn on personal results,
> 
> * It worked for me!
Every time I enter a device name and tern it on I go back and nothing happened and when I try to do it again it did nothing

**Comment by kvsbrnr** [comment]
> > Step 1: open google home. Step 2: go to news section. Step 3: ask for personal results on the device you installed the google client. Step 4: click on allow and name the device. Step 5: click on save and turn on personal results,
> > 
> > * It worked for me!
> >   Every time I enter a device name and tern it on I go back and nothing happened and when I try to do it again it did nothing

Same here. Nothing happens after naming device and enabling personal results. Google Assistant still asks to enable Personal Results

**Comment by sumeshir26** [comment]
When it asks, click the notification

**Comment by kvsbrnr** [comment]
Yeah clicking the notification redirects us to the page where we name device and enable Personal Results. After naming device and enabling personal results, nothing happens and Google assistant still asks to enable Personal Results.

**Comment by Ahmed00010** [comment]
> Yeah clicking the notification redirects us to the page where we name the device and enable Personal Results. There was no save option for me though. After naming the device and enabling personal results, I press back and try again. But nothing happens and Google assistant still asks to enable Personal Results.
same on Linux mint
anything yet?

**Comment by mattwhibley** [comment]
Same issue here. Attempted activation on Android, iOS and iPad OS. Notification appears, can be opened, but no save button is visible.

**Comment by brmonaghan** [comment]
same here

**Comment by sibbl** [comment]
Where is there supposed to be a save button? I only get this and cannot save anything: ![Screenshot](https://user-images.githubusercontent.com/866535/169521053-d76f1985-21e7-402f-967a-7d1ba8a8ce15.png)

**Comment by Abtaby** [comment]
> Where is there supposed to be a save button (sorry for German)? I only get this and cannot save anything. The empty space below the device name lets me enter the devices location but it also doesn't help. The other empty space with the X to the right lets me unlink the device.
> 
> ![Screenshot](https://user-images.githubusercontent.com/866535/169521053-d76f1985-21e7-402f-967a-7d1ba8a8ce15.png)
> 
> I set up my Assistant device of type phone and otherwise followed the tutorial steps.

Pls try using an Older version.

**Comment by Fedeli92** [comment]
> > Where is there supposed to be a save button (sorry for German)? I only get this and cannot save anything. The empty space below the device name lets me enter the devices location but it also doesn't help. The other empty space with the X to the right lets me unlink the device.
> > ![Screenshot](https://user-images.githubusercontent.com/866535/169521053-d76f1985-21e7-402f-967a-7d1ba8a8ce15.png)
> > I set up my Assistant device of type phone and otherwise followed the tutorial steps.
> 
> Pls try using an Older version of the home app.

Same here. No OK button to confirm name and and the toogle.
Google Home version 2.54.1.7 and Android 12

**Comment by Abtaby** [comment]
> > > Where is there supposed to be a save button (sorry for German)? I only get this and cannot save anything. The empty space below the device name lets me enter the devices location but it also doesn't help. The other empty space with the X to the right lets me unlink the device.
> > > ![Screenshot](https://user-images.githubusercontent.com/866535/169521053-d76f1985-21e7-402f-967a-7d1ba8a8ce15.png)
> > > I set up my Assistant device of type phone and otherwise followed the tutorial steps.
> > 
> > Pls try using an Older version of the home app.
> 
> Same here. No OK button to confirm name and and the toogle.
> Google Home version 2.54.1.7 and Android 12

Try this:
Just click on your account logo on the top-left, Then tap on 'Assistant Settings', Personal results, Other devices and tap on the device name and done! (Note: If device saved in a home, click on that home and tap the device name). It may show no name, but this still works without any name!

**Comment by praveshgupta1993** [comment]
I am also able to receive notification on mobile to Allow Personalized Results but clicking the notification doesn't show anything. It just keeps on repeating.

**Comment by praveshgupta1993** [comment]
> > Step 1: open google home. Step 2: go to news section. Step 3: ask for personal results on the device you installed the google client. Step 4: click on allow and name the device. Step 5: click on save and turn on personal results,
> > 
> > * It worked for me!
> 
> This worked for me. It might be an idea to add that going to the news section may be required because I didn't get an actual notification that the new device was asking for permission. I had to open the news section and then it was right at the top. Clicked (tapped) allow and then got the screen that was described in the set-up guide on this GitHub

I tried this as well, but didn't got the screen for device registration.

**Comment by praveshgupta1993** [comment]
> **Step 1**: Open google home. **Step 2**: Go (Tap) on the news section. **Step 3**: Ask for personal results on the device you installed the google client. **Step 4**: Click on allow and name the device. **Step 5**: Click on save and turn on personal results,
> 
> * It worked for me!

This didn't work for me.

**Comment by kuriandungu** [comment]
On Pixel6Pro with Android 13 the Allow notification comes after you ask assistant you name on pc. however tapping it crashes the app on phone immediately. Trying on an older device with Android 7 opens the dialog. you enter device name and enable personalized results however it doesnt save.

**Comment by EgidioCaprino** [comment]
> On Pixel6Pro with Android 13 the Allow notification comes after you ask assistant you name on pc. however tapping it crashes the app on phone immediately. Trying on an older device with Android 7 opens the dialog. you enter device name and enable personalized results however it doesn't save. I have also done the account linking procedure in Action Console but no joy still

Same problem here. Tried everything in this issue but it doesn't save the new device
